CHANGES.rst

```diff
@@ -403,6 +403,9 @@ astropy.stats
 astropy.table
 ^^^^^^^^^^^^^
 
+- Fixed a bug when a column in ``QTable`` is converted to ``Quantity``,
```

Comment:
This can be backported to the LTS 4.0.4 version, so it should go in the 4.0.4 section.